intezer_sdk/consts.py

```diff
@@ -19,6 +19,7 @@ class AnalysisStatusCode(enum.Enum):
 
 
 class AlertStatusCode(AutoName):
+    QUEUED = enum.auto()
```

Comment:
Do we actually have this?

Reply:
yes, before the submit message is start proccesing